tests/test_connection.py

```diff
@@ -37,6 +37,46 @@ def test_connect_bad(conn_cls):
         conn_cls.connect("dbname=nosuchdb")
 
 
+def test_connect_error_single_host_original_message_preserved(conn_cls):
+    with pytest.raises(psycopg.OperationalError) as e:
+        conn_cls.connect("dbname=nosuchdb")
+
+    msg = str(e)
+    assert 'database "nosuchdb" does not exist' in msg
+    assert "Multiple connection attempts failed" not in msg
```

Comment:
Please put this string in a const and use the same const for both the tests, otherwise, if we change the string this test would become invalid (it would pass even if we break something).

Reply:
Addressed in [982dda4](https://github.com/psycopg/psycopg/commit/982dda4f5c99832d95e0e447ba6c16eb9122a24a)